packages/next/build/webpack/config/blocks/css/index.ts

```diff
@@ -173,46 +218,62 @@ export const css = curry(async function css(
             exclude: /node_modules/,
           },
 
-          use: ([
-            // Add appropriate development more or production mode style
-            // loader
-            ctx.isClient &&
-              getClientStyleLoader({
-                isDevelopment: ctx.isDevelopment,
-                assetPrefix: ctx.assetPrefix,
-              }),
-
-            // Resolve CSS `@import`s and `url()`s
+          use: getModuleStyleLoader(
             {
-              loader: require.resolve('css-loader'),
-              options: {
-                importLoaders: 1,
-                sourceMap: true,
-                onlyLocals: ctx.isServer,
-                modules: {
-                  // Disallow global style exports so we can code-split CSS and
-                  // not worry about loading order.
-                  mode: 'pure',
-                  // Generate a friendly production-ready name so it's
-                  // reasonably understandable. The same name is used for
-                  // development.
-                  // TODO: Consider making production reduce this to a single
-                  // character?
-                  getLocalIdent: getCssModuleLocalIdent,
-                },
+              importLoaders: 1,
+              sourceMap: true,
+              onlyLocals: ctx.isServer,
+              modules: {
+                // Disallow global style exports so we can code-split CSS and
+                // not worry about loading order.
+                mode: 'pure',
+                // Generate a friendly production-ready name so it's
+                // reasonably understandable. The same name is used for
+                // development.
+                // TODO: Consider making production reduce this to a single
+                // character?
+                getLocalIdent: getCssModuleLocalIdent,
               },
             },
+            'sass-loader'
```

Comment:
This must be resolved.

Reply:
Is resolved in the loader compiling function https://github.com/zeit/next.js/pull/10133/files#diff-0df73c8e15d5a4ab2409907c614af8ebR170

Reply:
This is an odd indirection. Let's resolve it here instead.